skyportal/tests/api/test_candidates.py

```diff
@@ -13,10 +13,10 @@ def test_candidate_existence(view_only_token, public_candidate):
     status = api('HEAD', f'candidates/{public_candidate.id}', token=view_only_token)
     assert status == 200
 
-    status = api(
+    status, *_ = api(
```

Comment:
```suggestion
    status, _ = api(
```
Just a two-element tuple returned by `api`

Reply:
What happens if it returns a scalar? or is that never supposed to happen?

Reply:
I just mean the naming convention for unused variables. It's fine as-is, but I think `*_` is typically used to indicate multiple unused values, whereas `_` is used for single values (in this case a dict)

Reply:
I think *_ is preferred here because if there are no additional outputs beyond the first then it is ok, and if there are multiple additional outputs it is also ok. 
If you put status, _ = ... it expects exactly two outputs.

Reply:
`api` always returns at most two outputs

Reply:
Yes, but if it returns only one, and you specify status, _ = ... then it doesn't work, I think...

Reply:
I propose that we always return two outputs, making it consistent for `HEAD` requests.

Reply:
Agreed